### PR TITLE
System changes to support AWS EC2 and Postgresql

### PR DESCRIPTION
**Pull request by mberkowski**
* Renames sidekiq systemd services to `sidekiq-geoportal.service`
* Swaps Postgresql for MySQL
* A lot of robots.txt changes
* Whenever gem scheduling job template changes to find the right Ruby

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@
     t.string "to_state", null: false
     t.text "metadata"
     t.integer "sort_key", null: false
-    t.bigint "solr_document_sidecar_id", null: false
+    t.integer "solr_document_sidecar_id", null: false
```
Comment (ewlarson) [inline db/schema.rb:89]: PG should speak bigint... was this change really necessary?

**Review by ewlarson — CHANGES_REQUESTED** [review]
Just curious about that bigint => integer change here. Was that a blocker?

File: db/schema.rb (hunk 1 above)
Comment (mberkowski) [reply db/schema.rb:89]: Indeed pg does know bigint, and the databases were created with bigint. I don't know what rails did here with this in schema.rb (I never know what rails does in schema.rb). I'll pull this out.
